plugins/keywords-generator/src/main/kotlin/org/jetbrains/dataframe/keywords/KeywordsGeneratorPlugin.kt

```diff
@@ -0,0 +1,44 @@
+package org.jetbrains.dataframe.keywords
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.tasks.SourceSet
+import org.gradle.api.tasks.SourceSetContainer
+import org.gradle.kotlin.dsl.get
+import org.gradle.kotlin.dsl.register
+import org.jetbrains.kotlinx.dataframe.BuildConfig
+import java.io.File
+
+abstract class KeywordsGeneratorPlugin : Plugin<Project> {
+
+    override fun apply(target: Project): Unit = with(target) {
+        // from https://kotlinlang.org/docs/whatsnew21.html#compiler-symbols-hidden-from-the-kotlin-gradle-plugin-api
+        val taskDependencyScope = configurations.create("keywordsGeneratorScope")
+        dependencies.add(taskDependencyScope.name, "$KOTLIN_COMPILER_EMBEDDABLE:$KOTLIN_COMPILER_VERSION")
+        val taskResolvableConfiguration = configurations.create("keywordGeneratorResolvable") {
```

Comment:
What is the purpose? I know there are resolvable configurations in gradle, but this one seems to duplicate taskDependencyScope without modifications?

Reply:
also not sure, but they do it in the example https://kotlinlang.org/docs/whatsnew21.html#compiler-symbols-hidden-from-the-kotlin-gradle-plugin-api so there must be a reason for it I guess

Reply:
It represents a `ResolvableConfiguration`, will fail in gradle 9.0 if we don't split it